lib/widgets/layouts/layout.dart

```diff
@@ -35,13 +36,16 @@ class LayoutState extends State<Layout> with TickerProviderStateMixin {
   var isPanelClose;
 
   Widget showWalletList(BuildContext context) {
+    String walletId = Locator.instance.get<ApiDatabase>()
+        .walletStorage.currentWallet.id;
     return MouseRegion(
         cursor: SystemMouseCursors.click,
         child: GestureDetector(
           child: Container(
             color: WitnetPallet.white,
             width: 30,
             height: 30,
+            child: IdenticonWidget(seed: walletId ?? '', size: 8),
```

Comment:
The walletId can't be null
```suggestion
            child: IdenticonWidget(seed: walletId , size: 8),
```